Cargo.toml

```diff
@@ -1,6 +1,26 @@
 [workspace]
-members = [
-    "garde_tests",
-    "garde_derive",
-    "garde",
-]
+members = ["garde", "garde_*"]
+
+[workspace.dependencies]
+# Workspace crates
+garde = { version = "0.10.0", path = "./garde" }
```

Comment:
I use [cargo-workspaces](https://github.com/pksunkara/cargo-workspaces) to publish new releases, and it currently doesn't support workspace inheritance when bumping versions (https://github.com/pksunkara/cargo-workspaces/issues/82). Could you revert just `garde` and `garde_derive` back to using version+path directly in the individual crates' `Cargo.toml` files?

Reply:
Does it work now?

Reply:
Yes, thanks!